src/components/HierarchicalFacets.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  CollapsibleLabel,
+  CollapsibleSection,
+  FacetsProvider,
+  FilterGroupProvider,
+  HierarchicalFacetDisplay,
+  HierarchicalFacetDisplayCssClasses
+} from './Filters';
+import { CompositionMethod } from '../hooks/useComposedCssClasses';
+import { StandardFacetsProps } from './StandardFacets';
+import { Fragment } from 'react';
+
+/**
+ * The CSS class interface for {@link HierarchicalFacets}.
+ *
+ * @public
+ */
+export interface HierarchicalFacetsCssClasses extends HierarchicalFacetDisplayCssClasses {
+  container?: string,
+  divider?: string
+}
+
+/**
+ * Props for the {@link HierarchicalFacets} component.
+ *
+ * @public
+ */
+export interface HierarchicalFacetsProps extends Omit<StandardFacetsProps, 'searchable'> {
+  /** The `DisplayableFacet` to render as a HierarchicalFacetDisplay. */
+  fieldIds: string[],
+  /** The delimiter for determining facet hierarchies, defaults to "\>". */
+  delimiter?: string,
+  /** The maximum number of options to render before displaying the "Show more/less" button. Defaults to 4. */
+  showMoreLimit?: number,
+  /** CSS classes for customizing the component styling. */
+  customCssClasses?: HierarchicalFacetsCssClasses,
+  /** {@inheritDoc CompositionMethod} */
+  cssCompositionMethod?: CompositionMethod
+}
+
+/**
+ * A component that displays hierarchical facets, in a tree level structure,
+ * applicable to the current vertical search.
+ *
+ * @param props - {@link HierarchicalFacetsProps}
+ * @returns A React component for facets
+ *
+ * @public
+ */
+export function HierarchicalFacets({
+  searchOnChange,
+  collapsible,
+  defaultExpanded,
+  excludeFieldIds = [],
```

Comment:
why do we need both `excludeFieldIds` and `fieldIds`?

Reply:
Per comment above with oshi, I will remove`excludeFieldIds` (inherited from extending `StandardFacetsProps`) and have includedFieldIds for hierarchical and numerical